error-messages/user.messages.js

```diff
@@ -7,6 +7,7 @@ const EMAIL_ERROR = 'EMAIL_ERROR';
 const INVALID_PERMISSIONS = 'INVALID_PERMISSIONS';
 const RECOVERY_ATTEMPTS_LIMIT_EXCEEDED = 'RECOVERY_ATTEMPTS_LIMIT_EXCEEDED';
```

Comment:
Please, rename it to `PASSWORD_RECOVERY_ATTEMPTS_LIMIT_EXCEEDED`

Reply:
renamed